openff/toolkit/topology/molecule.py

```diff
@@ -4743,6 +4744,14 @@ def from_file(
                 file_format = file_path.split(".")[-1]
         file_format = file_format.upper()
 
+        if file_format == "XYZ":
+            raise UnsupportedFileTypeError(
+                "Parsing `.xyz` files is not currently supported because they lack sufficient "
+                "chemical information to be used with SMIRNOFF force fields. For more information, "
+                "see https://open-forcefield-toolkit.readthedocs.io/en/latest/faq.html or to provide "
```

Comment:
This redirects to the 0.10.1 build, which is fine, but will be slightly updated in the new release that includes #1139